client/src/app/core/repositories/motions/motion-repository.service.ts

```diff
@@ -271,7 +271,15 @@ export class MotionRepositoryService extends BaseIsAgendaItemAndListOfSpeakersCo
     };
 
     public getAgendaListTitle = (titleInformation: MotionTitleInformation) => {
-        const numberPrefix = titleInformation.agenda_item_number() ? `${titleInformation.agenda_item_number()} · ` : '';
+        let numberPrefix = '';
+        /**
+         * after reloading, titleInformation.agenda_item_number is undefined. It gets defined after
+         * an autoupdate
+         */
+        if (titleInformation.agenda_item_number && titleInformation.agenda_item_number()) {
```

Comment:
@FinnStutzenstein that one is pretty odd. Apparently, something that creates the motion restrictions does not deliver all information at a time, so agenda_item_number is undefined first, but was filled later. The error is not apparent. Ironically, using this conditions the view part works just fine